chains/solana/contracts/programs/ccip-router/src/instructions/v1/onramp.rs

```diff
@@ -306,13 +310,21 @@ fn token_transfer(
 // Helpers //
 /////////////
 
+// ProcessedExtraArgs contains serialized extra args and unpacks commonly used parameters
+#[derive(Debug)]
+pub struct ProcessedExtraArgs {
```

Comment:
Is this only for EVM should it declare it in the type?

Reply:
no it's chain agnostic - it has the serialized bytes, and two parameters that are checked for all chains (gas_limit + ooo)